src/extension.ts

```diff
@@ -263,7 +264,11 @@ export class Extension {
       const model = new TestModel(this._vscode, this._playwrightTest, workspaceFolderPath, configFileUri.fsPath, playwrightInfo, this._envProvider.bind(this));
       this._models.push(model);
       this._testTree.addModel(model);
-      await model.listFiles();
+      const configError = await model.listFiles();
+      if (configError) {
+        configErrors.set(configError.location?.file ?? configFilePath, configError);
```

Comment:
I would untangle config errors from other errors and report them separately, in a separate set of decorations, with its own lifetime.